### PR TITLE
V5 migration

### PR DESCRIPTION
**Pull request by jsernaque**

### CODE REVIEW

**Comment by tristanlee85** [comment]
FYI https://github.com/layer0-docs/layer0-docs/pull/452

File: constants.js [hunk 1]
```diff
@@ -7,10 +7,11 @@ const PRODUCT_PLATFORM = 'App Platform';
...
-const PACKAGE_NAME = '@layer0';
-const CONFIG_FILE = 'layer0.config.js';
+
+const CLI_NAME = 'edg';
+const FULL_CLI_NAME = 'edgio';
+const PACKAGE_NAME = '@edgio';
+const CONFIG_FILE = 'edgio.config.js';
 const HEADER_PREFIX = 'x-0';
 const COOKIE_PREFIX = 'layer0';
 const RUM_NS = 'Layer0'; // namespace for the JS package used by RUM
```
Comment (petrvecera) [inline constants.js:17]: How is this used? I think we should update this to `edgio` https://github.com/layer0-platform/rum/pull/26/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 ?

Comment (tristanlee85) [reply constants.js:17]: Does `edgio` command have a short alias? `edg` should be whatever is the equivalent of `0` currently.

Comment (petrvecera) [reply constants.js:17]: Yes, `edg` is equivalent of `0` and full command is `edgio`.

File: src/pages/guides/v5_migration.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+For each site, run the following command:
+
+`{{ FULL_CLI_NAME }} init`
+
+Proceed to the next step if this command is successful.
+
+## Step 5: Update Scripts that Reference the {{ PRODUCT }} CLI {/*step-5-update-scripts-that-reference-the-cli*/}
+
+Update all references to the {{ PRODUCT }} CLI within your scripts from `0 | layer0` to either `{{ FULL_CLI_NAME }}` or `{{ FULL_CLI_NAME }}`.
```
Comment (petrvecera) [inline src/pages/guides/v5_migration.md:127]:
```suggestion
Update all references to the {{ PRODUCT }} CLI within your scripts from `0 | layer0` to either `{{ CLI_NAME }}` or `{{ FULL_CLI_NAME }}`.
```

**Review by tristanlee85 — APPROVED** [review]

**Pull request merged**